### ISSUE TITLE
Link to Azure DevOps Server labs missing

### ISSUE THREAD

**Issue by devopsjester** [microsoft/azuredevopslabs#412]
I just noticed that the landing page for https://azuredevopslabs.com no longer has a link to its server counterpart, and the same holds for the server labs page. This makes the server labs very difficult to find, unnecessarily so.

Is this intentional, or a defect – in which case, is there a timeframe for when it would be fixed?

**Comment by sriramdasbalaji** [comment]
Hi @devopsjester ,

Server labs available at https://azuredevopslabs.com/labs/devopsserver/
But LabOnDemand VMs are not available to practices. You need to spin up your own VM as explained in 
https://azuredevopslabs.com/labs/devopsserver/devopsvmdownload/

Thanks

**Comment by devopsjester** [comment]
Hi. 

Thanks for the clarification. After some searching, I found the Azure DevOps labs for server myself. 

My point was that in the past, there used to be a link from the services version to the server version and vice versa. 

Now there isn’t. I had to use an external search engine to find it. 

That makes finding the other (usually server) flavor unnecessarily difficult. 

Shouldn’t that link be returned?
